custom-recipes/nlp-visualization-wordcloud/recipe.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import logging
+from time import perf_counter
+
+from dataiku.customrecipe import get_recipe_resource
+from spacy_tokenizer import MultilingualTokenizer
+from wordcloud_visualizer import WordcloudVisualizer
+from plugin_config_loading import load_plugin_config_wordcloud
+
+
+# Load config
+params = load_plugin_config_wordcloud()
+resource_path = get_recipe_resource()
+output_folder = params["output_folder"]
+df = params["df"]
+
+# Load tokenizer
+tokenizer = MultilingualTokenizer()
+
+# Load wordcloud visualizer
+worcloud_visualizer = WordcloudVisualizer(
+    tokenizer=tokenizer,
+    text_column=params["text_column"],
+    font_path=resource_path,
+    language=params["language"],
+    language_column=params["language_column"],
+    subchart_column=params["subchart_column"],
+)
+
+# Prepare data and count tokens for each subchart
+frequencies = worcloud_visualizer.prepare_and_count(df)
```

Comment:
Small naming nitpick, I believe this
```suggestion
frequencies = worcloud_visualizer.tokenize_and_count(df)
```
would describe what the method does more appropriately